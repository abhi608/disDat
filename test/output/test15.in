Starting T5
Starting T4
Starting T3
Starting T2
Starting T1
T1 got write lock on x4
Site: 2 failed
T1 aborted as site 2 failed
DEBUG| Clearing site locks for T1 variable: x4
Clearing locks for T1 variable: x4
T2 got write lock on x4
T2 got read lock on x4 having value 44
Site: 2 recovered
DEBUG| T3 did not get write lock on x4 site: 1
DEBUG| T3 did not get write lock on x4 site: 3
DEBUG| T3 did not get write lock on x4 site: 4
DEBUG| T3 did not get write lock on x4 site: 5
DEBUG| T3 did not get write lock on x4 site: 6
DEBUG| T3 did not get write lock on x4 site: 7
DEBUG| T3 did not get write lock on x4 site: 8
DEBUG| T3 did not get write lock on x4 site: 9
DEBUG| T3 did not get write lock on x4 site: 10
T3 is blocked for a write lock by T2 on x4
DEBUG| T4 did not get write lock on x4 site: 1
DEBUG| T4 did not get write lock on x4 site: 2
DEBUG| T4 did not get write lock on x4 site: 3
DEBUG| T4 did not get write lock on x4 site: 4
DEBUG| T4 did not get write lock on x4 site: 5
DEBUG| T4 did not get write lock on x4 site: 6
DEBUG| T4 did not get write lock on x4 site: 7
DEBUG| T4 did not get write lock on x4 site: 8
DEBUG| T4 did not get write lock on x4 site: 9
DEBUG| T4 did not get write lock on x4 site: 10
T4 is blocked for a write lock by T2 on x4
T4 is blocked for a write lock by T3 on x4
DEBUG| T5 did not get write lock on x4 site: 1
DEBUG| T5 did not get write lock on x4 site: 2
DEBUG| T5 did not get write lock on x4 site: 3
DEBUG| T5 did not get write lock on x4 site: 4
DEBUG| T5 did not get write lock on x4 site: 5
DEBUG| T5 did not get write lock on x4 site: 6
DEBUG| T5 did not get write lock on x4 site: 7
DEBUG| T5 did not get write lock on x4 site: 8
DEBUG| T5 did not get write lock on x4 site: 9
DEBUG| T5 did not get write lock on x4 site: 10
T5 is blocked for a write lock by T2 on x4
T5 is blocked for a write lock by T3 on x4
T2 read the value 44 of variable x4
T2 committed
DEBUG| Clearing site locks for T2 variable: x4
Clearing locks for T2 variable: x4
DEBUG| T5 did not get write lock on x4 site: 2
T5 is blocked for a write lock by T3 on x4
DEBUG| T4 did not get write lock on x4 site: 1
DEBUG| T4 did not get write lock on x4 site: 2
DEBUG| T4 did not get write lock on x4 site: 3
DEBUG| T4 did not get write lock on x4 site: 4
DEBUG| T4 did not get write lock on x4 site: 5
DEBUG| T4 did not get write lock on x4 site: 6
DEBUG| T4 did not get write lock on x4 site: 7
DEBUG| T4 did not get write lock on x4 site: 8
DEBUG| T4 did not get write lock on x4 site: 9
DEBUG| T4 did not get write lock on x4 site: 10
T4 is blocked for a write lock by T5 on x4
T4 is blocked for a write lock by T3 on x4
DEBUG| T3 did not get write lock on x4 site: 1
DEBUG| T3 did not get write lock on x4 site: 3
DEBUG| T3 did not get write lock on x4 site: 4
DEBUG| T3 did not get write lock on x4 site: 5
DEBUG| T3 did not get write lock on x4 site: 6
DEBUG| T3 did not get write lock on x4 site: 7
DEBUG| T3 did not get write lock on x4 site: 8
DEBUG| T3 did not get write lock on x4 site: 9
DEBUG| T3 did not get write lock on x4 site: 10
T3 is blocked for a write lock by T5 on x4
T3 aborted as it is youngest in a deadlock
DEBUG| Clearing site locks for T3 variable: x4
T5 got write lock on x4
DEBUG| T4 did not get write lock on x4 site: 1
DEBUG| T4 did not get write lock on x4 site: 2
DEBUG| T4 did not get write lock on x4 site: 3
DEBUG| T4 did not get write lock on x4 site: 4
DEBUG| T4 did not get write lock on x4 site: 5
DEBUG| T4 did not get write lock on x4 site: 6
DEBUG| T4 did not get write lock on x4 site: 7
DEBUG| T4 did not get write lock on x4 site: 8
DEBUG| T4 did not get write lock on x4 site: 9
DEBUG| T4 did not get write lock on x4 site: 10
T4 is blocked for a write lock by T5 on x4
DEBUG| T4 did not get write lock on x4 site: 1
DEBUG| T4 did not get write lock on x4 site: 2
DEBUG| T4 did not get write lock on x4 site: 3
DEBUG| T4 did not get write lock on x4 site: 4
DEBUG| T4 did not get write lock on x4 site: 5
DEBUG| T4 did not get write lock on x4 site: 6
DEBUG| T4 did not get write lock on x4 site: 7
DEBUG| T4 did not get write lock on x4 site: 8
DEBUG| T4 did not get write lock on x4 site: 9
DEBUG| T4 did not get write lock on x4 site: 10
T4 committed
T5 committed
DEBUG| Clearing site locks for T5 variable: x4
Clearing locks for T5 variable: x4
=== Site 1 ===
x4: 77 at site 1
All other variables have same initial value
=== Site 2 ===
x20 is not available for reading
x18 is not available for reading
x14 is not available for reading
x16 is not available for reading
x12 is not available for reading
x11: 110 (available at site 2 for reading as it is the only copy or has been written after recovery)
x10 is not available for reading
x4: 77 (available at site 2 for reading as it is the only copy or has been written after recovery)
x1: 10 (available at site 2 for reading as it is the only copy or has been written after recovery)
x2 is not available for reading
x8 is not available for reading
x6 is not available for reading
=== Site 3 ===
x4: 77 at site 3
All other variables have same initial value
=== Site 4 ===
x4: 77 at site 4
All other variables have same initial value
=== Site 5 ===
x4: 77 at site 5
All other variables have same initial value
=== Site 6 ===
x4: 77 at site 6
All other variables have same initial value
=== Site 7 ===
x4: 77 at site 7
All other variables have same initial value
=== Site 8 ===
x4: 77 at site 8
All other variables have same initial value
=== Site 9 ===
x4: 77 at site 9
All other variables have same initial value
=== Site 10 ===
x4: 77 at site 10
All other variables have same initial value
