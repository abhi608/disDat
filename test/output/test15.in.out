Starting T5
Starting T4
Starting T3
Starting T2
Starting T1
T1 got write lock on x4
Site: 2 failed
T1 aborted as site 2 failed
DEBUG| Clearing site locks for T1 variable: x4
Clearing locks for T1 variable: x4
T2 got write lock on x4
T2 got read lock on x4 having value 44
Site: 2 recovered
DEBUG| T3 did not get write lock on x4 site: 1
DEBUG| T3 did not get write lock on x4 site: 3
DEBUG| T3 did not get write lock on x4 site: 4
DEBUG| T3 did not get write lock on x4 site: 5
DEBUG| T3 did not get write lock on x4 site: 6
DEBUG| T3 did not get write lock on x4 site: 7
DEBUG| T3 did not get write lock on x4 site: 8
DEBUG| T3 did not get write lock on x4 site: 9
DEBUG| T3 did not get write lock on x4 site: 10
T3 is blocked for a write lock by T2 on x4
DEBUG| T4 did not get write lock on x4 site: 1
DEBUG| T4 did not get write lock on x4 site: 2
DEBUG| T4 did not get write lock on x4 site: 3
DEBUG| T4 did not get write lock on x4 site: 4
DEBUG| T4 did not get write lock on x4 site: 5
DEBUG| T4 did not get write lock on x4 site: 6
DEBUG| T4 did not get write lock on x4 site: 7
DEBUG| T4 did not get write lock on x4 site: 8
DEBUG| T4 did not get write lock on x4 site: 9
DEBUG| T4 did not get write lock on x4 site: 10
T4 is blocked for a write lock by T2 on x4
T4 is blocked for a write lock by T3 on x4
DEBUG| T5 did not get write lock on x4 site: 1
DEBUG| T5 did not get write lock on x4 site: 2
DEBUG| T5 did not get write lock on x4 site: 3
DEBUG| T5 did not get write lock on x4 site: 4
DEBUG| T5 did not get write lock on x4 site: 5
DEBUG| T5 did not get write lock on x4 site: 6
DEBUG| T5 did not get write lock on x4 site: 7
DEBUG| T5 did not get write lock on x4 site: 8
DEBUG| T5 did not get write lock on x4 site: 9
DEBUG| T5 did not get write lock on x4 site: 10
T5 is blocked for a write lock by T2 on x4
T5 is blocked for a write lock by T3 on x4
T2 read the value 44 of variable x4
T2 write the value 44 of variable x4
T2 write the value 44 of variable x4
T2 write the value 44 of variable x4
T2 write the value 44 of variable x4
T2 write the value 44 of variable x4
T2 write the value 44 of variable x4
T2 write the value 44 of variable x4
T2 write the value 44 of variable x4
T2 write the value 44 of variable x4
T2 committed
DEBUG| Clearing site locks for T2 variable: x4
Clearing locks for T2 variable: x4
DEBUG| T5 did not get write lock on x4 site: 2
T5 is blocked for a write lock by T3 on x4
DEBUG| T4 did not get write lock on x4 site: 1
DEBUG| T4 did not get write lock on x4 site: 2
DEBUG| T4 did not get write lock on x4 site: 3
DEBUG| T4 did not get write lock on x4 site: 4
DEBUG| T4 did not get write lock on x4 site: 5
DEBUG| T4 did not get write lock on x4 site: 6
DEBUG| T4 did not get write lock on x4 site: 7
DEBUG| T4 did not get write lock on x4 site: 8
DEBUG| T4 did not get write lock on x4 site: 9
DEBUG| T4 did not get write lock on x4 site: 10
T4 is blocked for a write lock by T5 on x4
T4 is blocked for a write lock by T3 on x4
DEBUG| T3 did not get write lock on x4 site: 1
DEBUG| T3 did not get write lock on x4 site: 3
DEBUG| T3 did not get write lock on x4 site: 4
DEBUG| T3 did not get write lock on x4 site: 5
DEBUG| T3 did not get write lock on x4 site: 6
DEBUG| T3 did not get write lock on x4 site: 7
DEBUG| T3 did not get write lock on x4 site: 8
DEBUG| T3 did not get write lock on x4 site: 9
DEBUG| T3 did not get write lock on x4 site: 10
T3 is blocked for a write lock by T5 on x4
T3 aborted as it is youngest in a deadlock
DEBUG| Clearing site locks for T3 variable: x4
T5 got write lock on x4
DEBUG| T4 did not get write lock on x4 site: 1
DEBUG| T4 did not get write lock on x4 site: 2
DEBUG| T4 did not get write lock on x4 site: 3
DEBUG| T4 did not get write lock on x4 site: 4
DEBUG| T4 did not get write lock on x4 site: 5
DEBUG| T4 did not get write lock on x4 site: 6
DEBUG| T4 did not get write lock on x4 site: 7
DEBUG| T4 did not get write lock on x4 site: 8
DEBUG| T4 did not get write lock on x4 site: 9
DEBUG| T4 did not get write lock on x4 site: 10
T4 is blocked for a write lock by T5 on x4
DEBUG| T4 did not get write lock on x4 site: 1
DEBUG| T4 did not get write lock on x4 site: 2
DEBUG| T4 did not get write lock on x4 site: 3
DEBUG| T4 did not get write lock on x4 site: 4
DEBUG| T4 did not get write lock on x4 site: 5
DEBUG| T4 did not get write lock on x4 site: 6
DEBUG| T4 did not get write lock on x4 site: 7
DEBUG| T4 did not get write lock on x4 site: 8
DEBUG| T4 did not get write lock on x4 site: 9
DEBUG| T4 did not get write lock on x4 site: 10
T4 committed
T5 write the value 77 of variable x4
T5 write the value 77 of variable x4
T5 write the value 77 of variable x4
T5 write the value 77 of variable x4
T5 write the value 77 of variable x4
T5 write the value 77 of variable x4
T5 write the value 77 of variable x4
T5 write the value 77 of variable x4
T5 write the value 77 of variable x4
T5 write the value 77 of variable x4
T5 committed
DEBUG| Clearing site locks for T5 variable: x4
Clearing locks for T5 variable: x4
=== Site 1 ===
x20: 200 at site 1
x16: 160 at site 1
x14: 140 at site 1
x10: 100 at site 1
x8: 80 at site 1
x6: 60 at site 1
x4: 77 at site 1
x18: 180 at site 1
x12: 120 at site 1
x2: 20 at site 1
=== Site 2 ===
x20 is not available for reading
x16 is not available for reading
x14 is not available for reading
x8 is not available for reading
x6 is not available for reading
x11: 110 (available at site 2 for reading as it is the only copy or has been written after recovery)
x4: 77 (available at site 2 for reading as it is the only copy or has been written after recovery)
x18 is not available for reading
x12 is not available for reading
x2 is not available for reading
x10 is not available for reading
x1: 10 (available at site 2 for reading as it is the only copy or has been written after recovery)
=== Site 3 ===
x20: 200 at site 3
x16: 160 at site 3
x14: 140 at site 3
x10: 100 at site 3
x8: 80 at site 3
x6: 60 at site 3
x4: 77 at site 3
x18: 180 at site 3
x12: 120 at site 3
x2: 20 at site 3
=== Site 4 ===
x20: 200 at site 4
x16: 160 at site 4
x14: 140 at site 4
x10: 100 at site 4
x6: 60 at site 4
x4: 77 at site 4
x13: 130 at site 4
x8: 80 at site 4
x3: 30 at site 4
x18: 180 at site 4
x12: 120 at site 4
x2: 20 at site 4
=== Site 5 ===
x20: 200 at site 5
x16: 160 at site 5
x14: 140 at site 5
x10: 100 at site 5
x8: 80 at site 5
x6: 60 at site 5
x4: 77 at site 5
x18: 180 at site 5
x12: 120 at site 5
x2: 20 at site 5
=== Site 6 ===
x20: 200 at site 6
x14: 140 at site 6
x15: 150 at site 6
x10: 100 at site 6
x8: 80 at site 6
x6: 60 at site 6
x16: 160 at site 6
x5: 50 at site 6
x4: 77 at site 6
x18: 180 at site 6
x12: 120 at site 6
x2: 20 at site 6
=== Site 7 ===
x20: 200 at site 7
x16: 160 at site 7
x14: 140 at site 7
x10: 100 at site 7
x8: 80 at site 7
x6: 60 at site 7
x4: 77 at site 7
x18: 180 at site 7
x12: 120 at site 7
x2: 20 at site 7
=== Site 8 ===
x17: 170 at site 8
x16: 160 at site 8
x14: 140 at site 8
x10: 100 at site 8
x8: 80 at site 8
x20: 200 at site 8
x7: 70 at site 8
x6: 60 at site 8
x4: 77 at site 8
x18: 180 at site 8
x12: 120 at site 8
x2: 20 at site 8
=== Site 9 ===
x20: 200 at site 9
x16: 160 at site 9
x14: 140 at site 9
x10: 100 at site 9
x8: 80 at site 9
x6: 60 at site 9
x4: 77 at site 9
x18: 180 at site 9
x12: 120 at site 9
x2: 20 at site 9
=== Site 10 ===
x16: 160 at site 10
x19: 190 at site 10
x14: 140 at site 10
x10: 100 at site 10
x20: 200 at site 10
x9: 90 at site 10
x8: 80 at site 10
x6: 60 at site 10
x4: 77 at site 10
x18: 180 at site 10
x12: 120 at site 10
x2: 20 at site 10
